src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -255,26 +255,19 @@ static void _applyComposition(Paint* paint, const SvgNode* node, float vx, float
         if (compNode && compNode->child.count > 0) {
             node->style->mask.applying = true;
 
-            auto comp = Shape::gen();
-            comp->fill(255, 255, 255, 255);
-            if (node->transform) comp->transform(*node->transform);
-
-            auto child = compNode->child.data;
-            auto valid = false; //Composite only when valid shapes are existed
-
-            for (uint32_t i = 0; i < compNode->child.count; ++i, ++child) {
-                if (_appendChildShape(*child, comp.get(), vx, vy, vw, vh)) valid = true;
+            auto comp = _sceneBuildHelper(compNode, vx, vy, vw, vh, svgPath, true);
+            if (comp) {
+                if (node->transform) comp->transform(*node->transform);
```

Comment:
(Sorry for not reviewing all at once.)
comp(scene) has already a transform matrix when there is transform information in the mask at line 564.
This code will overwrite the transform information.
You'll probably have to use matrix multiply.

ex)
```html
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"  viewbox="-10 -10 120 120"> 
  <mask id="mymask" transform="translate(50 0)">
    <rect x="0" y="0" width="50" height="50" fill="red" />
    <rect x="50" y="50" width="50" height="50" fill="blue" />  <!-- red changed to blue -->
  </mask>

  <polygon points="-10,110 110,110 110,-10" fill="orange" />
  <circle cx="50" cy="50" r="50" mask="url(#mymask)" transform="translate(-25 0)" fill="red"/>
</svg>
```

![image](https://user-images.githubusercontent.com/7413838/140287141-a9735759-ca4d-40cb-a43d-90e5c271ad91.png)

Reply:
@JSUYA thank you for such a detailed rev!
I changed the code, but in a different way than you asked, since according to the standard a mask can not contain transform attribute (and a clipPath can :) https://www.w3.org/TR/SVG11/masking.html)
so in this case overwriting was good :D
so if we build a scene that is a mask, the transformation is skipped now

Reply:
so the result from the image viewer is incorrect :)

Reply:
oh thank you for check.
I often use this site to check the specification. https://developer.mozilla.org/en-US/docs/Web/SVG/Element/mask
There is a transform in global attribute in this url, so I thought we should support transform.
anyway, ubuntu image viewer is incorrect. :)